tools/detection2d/utils.py

```diff
@@ -0,0 +1,49 @@
+import re
```

Comment:
Please do not place standalone algorithm scripts such as `tools/detection2d/deploy_yolox_s_opt.py` or items like those in `tools/detection2d` under the tools directory.
If the tools only use for `YOLOX`, would you move into `projects/YOLOX_elan`?

Reply:
They are moved into projects: https://github.com/tier4/AWML/pull/46/commits/88a3c8a338e1b6cc35eb61cc0ea7ea05d8d7ba04
https://github.com/tier4/AWML/pull/48/commits/8349808bda07cc356f9b321a28e62be9fdd04084